packages/block-editor/src/components/block-toolbar/style.scss

```diff
@@ -46,15 +49,18 @@
 			border-color: $light-gray-500;
 		}
 	}
+}
 
-	&__inserter {
-		opacity: 0;
-		pointer-events: none;
+// Inserter in the toolbar for tabbing into.
+.block-editor-block-toolbar__inserter {
+	opacity: 0;
+	pointer-events: none;
+	width: 0;
 
-		&.is-visible {
-			opacity: 1;
-			pointer-events: all;
-		}
+	&.is-visible {
+		width: auto;
+		opacity: 1;
+		pointer-events: all;
 	}
 }
```

Comment:
@ellatrix I feel the inserter shouldn't be part of the BlockToolbar DOM node at all, it makes styling more difficult. Could we return maybe a fragment and move it out?

Reply:
Sure. I moved it here temporarily. We can move it somewhere else and reposition.